sound/soc/sof/intel/hda.c

```diff
@@ -1476,7 +1476,7 @@ struct sof_hda_ssid_quirk {
 };
 
 static const struct sof_hda_ssid_quirk sof_hda_ssid_quirks[] = {
-	{ PCI_VENDOR_ID_LENOVO, 0x3906, "aw88399" },
+	/* Empty - AW88399 uses dynamic SSP detection instead of SSID quirk */
```

Comment:
Maybe remove `sof_hda_ssid_quirks[]` completely and squash the commit to the previous commit.